src/viam/robot/client.py

```diff
@@ -262,7 +262,7 @@ async def _check_connection(self, check_every: int, reconnect_every: int):
 
             # Failure to grab resources could be for spurious, non-networking reasons. Try three times just to be safe.
             connection_error = None
-            for attempt in range(3):
+            for _ in range(3):
```

Comment:
I added this to mirror the loop structures since this `attempt` iterator is unused anyways